### ISSUE TITLE
[Bug] LED Control Does Not Work

### ISSUE THREAD

**Issue by mtl010957** [Vaskivskyi/ha-asusrouter#710]
### The problem

The LED control switch no longer works, throws an error in the logs and on the GUI just returns to whatever position it was last in.

### Your device model

RT-AX58U

### Firmware type

Merlin

### Firmware version

3004.388.4

### Integration version with the issue

0.24.0b2

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

Core 2023.11.2

### Is there anything useful in the logs?

```txt
This error originated from a custom integration.

Logger: custom_components.asusrouter.entity
Source: custom_components/asusrouter/entity.py:168
Integration: AsusRouter (documentation, issues)
First occurred: 9:43:08 AM (1 occurrences)
Last logged: 9:43:08 AM

Unable to set state with an exception: set_state() takes from 2 to 4 positional arguments but 5 were given
```

### Additional information

LED Control works as expected from both the Android app and the web admin interface.

**Comment by Vaskivskyi** [comment]
This will be fixed as soon as possible

**Comment by Vaskivskyi** [comment]
The issue should be fixed in `0.24.0b3`

**Comment by mtl010957** [comment]
Thanks, I confirmed this is fixed in 0.24.0b3!
